### DISCUSSION TITLE
Why do my single quotation marks occupy too wide space?

### DISCUSSION THREAD

**Discussion by ZiwuValley** [ppoffice/hexo-theme-icarus#1256] [Q&A]
I built a blog and posted an article on it. However I found that the single quotation marks in the article occupy too wide space like the following picture.

![image](https://github.com/ppoffice/hexo-theme-icarus/assets/154729300/43eb2f92-ae36-4f12-ab70-af8aca8d6249)

It seems that quotation marks in the official demo do so.

![image](https://github.com/ppoffice/hexo-theme-icarus/assets/154729300/c5868773-21ed-4545-8473-a49a2ff07871)

How should I fix it?

**Comment by ppoffice** [comment]
@ZiwuValley What browser are you using? Can you turn off all extensions and try again? Or try with another browser?

**Reply by ZiwuValley** [reply]
I'm using Microsoft Edge as my browser. And I turned off all extensions but it didn't work. 

But the quotation marks display properly on my cell phone.

**Comment by ZiwuValley** [comment]
![image](https://github.com/ppoffice/hexo-theme-icarus/assets/154729300/d7bc1695-db34-48c8-a640-ff5079817340)

I guess this error cause the problem I met. How should I fix it?

**Comment by ZiwuValley** [comment]
I replaced the fonts with https://fonts.googlefonts.cn/css?family=Ubuntu and I solved this problem. Thanks.
